src/components/pipelineRuns-list/PipelineRunsList.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  ListPageBody,
+  ListPageFilter,
+  VirtualizedTable,
+  getGroupVersionKindForModel,
+  useK8sWatchResource,
+  useListPageFilter,
+} from '@openshift-console/dynamic-plugin-sdk';
+import * as React from 'react';
+import usePipelineRunsColumns from './usePipelineRunsColumns';
+import { usePipelineRunsFilters } from './usePipelineRunsFilters';
+import { PipelineRunKind } from '../../types';
+import { PipelineRunModel } from '../../models';
+import { useGetTaskRuns } from '../hooks/useTektonResult';
+import PipelineRunsRow from './PipelineRunsRow';
+import { useTranslation } from 'react-i18next';
+
+import './PipelineRunsList.scss';
+import { useParams } from 'react-router-dom-v5-compat';
+
+type PipelineRunsListProps = {
+  namespace: string;
+};
+
+const PipelineRunsList: React.FC<PipelineRunsListProps> = ({ namespace }) => {
+  const { t } = useTranslation();
+  const { ns } = useParams();
+  namespace = namespace || ns;
+  const columns = usePipelineRunsColumns(namespace);
+  const filters = usePipelineRunsFilters();
+  const [pipelineRuns, pipelineRunsLoaded, pipelineRunsLoadError] =
+    useK8sWatchResource<PipelineRunKind[]>({
+      isList: true,
+      groupVersionKind: getGroupVersionKindForModel(PipelineRunModel),
+      namespace,
+      optional: true,
+    });
+  const [data, filteredData, onFilterChange] = useListPageFilter(
+    pipelineRuns,
+    filters,
+  );
+  const [taskRuns] = useGetTaskRuns(namespace);
+  return (
+    <ListPageBody>
+      <ListPageFilter
+        columnLayout={{
+          columns: columns?.map(({ id, title }) => ({ id, title })),
+          id: 'pipelineRuns-list',
+          type: 'PipelineRun',
+          selectedColumns: new Set(['name']),
+        }}
+        rowFilters={filters}
+        onFilterChange={onFilterChange}
+        data={data}
+        loaded={pipelineRunsLoaded}
+        hideColumnManagement
+      />
+      <VirtualizedTable<PipelineRunKind>
+        EmptyMsg={() => (
+          <div className="pf-u-text-align-center" id="no-templates-msg">
+            {t('No PipelineRuns found')}
+          </div>
+        )}
```

Comment:
<img width="1439" alt="Screenshot 2024-03-25 at 10 36 16 AM" src="https://github.com/openshift-pipelines/console-plugin/assets/102503482/8fc5c7d2-97a2-4111-ba48-708732ff4ac0">

Padding is missing for empty PLR message